cylc/flow/scheduler.py

```diff
@@ -441,7 +443,11 @@ async def configure(self):
                 pri_dao.close()
 
         self.profiler.log_memory("scheduler.py: before load_flow_file")
-        self.load_flow_file()
+        try:
+            self.load_flow_file()
+        except ParsecError as exc:
+            exc.schd_expected = True
```

Comment:
Could you add a comment explaining what the `exc.schd_expected` is about. Not so keen on the name `schd_expected` but can't think of anything better.